x-pack/legacy/plugins/infra/public/pages/logs/log_entry_categories/sections/top_categories/top_categories_section.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+import { LogEntryCategory } from '../../../../../../common/http_api/log_analysis';
+import { TimeRange } from '../../../../../../common/http_api/shared';
+import { LoadingOverlayWrapper } from '../../../../../components/loading_overlay_wrapper';
+import { RecreateJobButton } from '../../../../../components/logging/log_analysis_job_status';
+import { AnalyzeInMlButton } from '../../../../../components/logging/log_analysis_results';
+import { DatasetsSelector } from './datasets_selector';
+import { TopCategoriesTable } from './top_categories_table';
+
+export const TopCategoriesSection: React.FunctionComponent<{
+  availableDatasets: string[];
+  isLoadingDatasets?: boolean;
+  isLoadingTopCategories?: boolean;
+  jobId: string;
+  onChangeDatasetSelection: (datasets: string[]) => void;
+  onRequestRecreateMlJob: () => void;
+  selectedDatasets: string[];
+  timeRange: TimeRange;
+  topCategories: LogEntryCategory[];
+}> = ({
+  availableDatasets,
+  isLoadingDatasets = false,
+  isLoadingTopCategories = false,
+  jobId,
+  onChangeDatasetSelection,
+  onRequestRecreateMlJob,
+  selectedDatasets,
+  timeRange,
+  topCategories,
+}) => {
+  return (
+    <>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <EuiTitle size="m" aria-label={title}>
+            <h2>{title}</h2>
```

Comment:
```suggestion
            <h1>{title}</h1>
```
We should have an H1 element on every view